bigbluebutton-html5/imports/ui/components/poll/component.jsx

```diff
@@ -192,6 +199,7 @@ class Poll extends Component {
   }
 
   renderInputFields() {
+    const MAX_INPUT_CHARS = 75;
```

Comment:
Could you move this constant to the top of the file. I also think that the max should be less. The rendering of 75 characters is pretty bad, but something like 40-50 is more readable.